src/util/lineGeometry.js

```diff
@@ -67,21 +67,22 @@
         //round to 45 degree increments
         var roundedAngle = (Math.round((lineAngle*4) / Math.PI)/4)*180;
 
-        var distance = (Math.abs(xdiff) + Math.abs(ydiff))/2;
+        //keep hypontenuse the same length even after changing angle
+        var distance = Math.sqrt((xdiff) * (xdiff) + (ydiff) * (ydiff));
```

Comment:
The funny thing is that there are a lot of parentheses but they don't actually clarify the order of operations. :grin: